airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/schemas/sponsored_display.py

```diff
@@ -48,6 +48,14 @@ class DisplayTargeting(Targeting):
     resolvedExpression: List[Dict[str, str]]
 
 
+class DisplayCreatives(CatalogModel):
+    adGroupId: Decimal
+    creativeId: Decimal
+    creativeType: str
+    properties: List[Dict[str, str]]
```

Comment:
According to the [API reference](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Creatives/operation/listCreatives), this should be
```suggestion
    properties: Dict[str, Any]
```

Reply:
Tests are not passing since the `Any` datatype isn't imported. Would it make sense to import it, or do you think I should declare specific types for all the nested fields? Some of the nested objects are several layers deep so it would be complex, but I think it's possible. ([api docs link](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Creatives/operation/listCreatives))

Reply:
I've declared the nested fields in https://github.com/airbytehq/airbyte/pull/26203/commits/6ad2e7cc9b7f275622409b4af93a4b3551d341a1 hopefully that works 🙏🏽

Reply:
@sh4sh sorry to disappoint you but that definition is not correct. It should be `Union` instead of a data class. On the other hand, our destinations do not support anyOf/allOf which is represented by Union, thus even Union is redundant. So, importing Any should be enough to declare generic object prroperty

Reply:
Ah well I tried. Thanks for the explanation, that makes sense! Reverted in https://github.com/airbytehq/airbyte/pull/26203/commits/9ebb00b4d8b36d6ed0f3dd6d576ffada6dde88db and imported `Any` in https://github.com/airbytehq/airbyte/pull/26203/commits/673cc3154d1dfbe86ab63dbbf4cebc88377eef43